lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -31,6 +48,18 @@
                     "methods": [
                         "CurationConcerns::Workflow::ActivateObject"
                     ]
+                }, {
+                    "name": "request_review",
+                    "from_states": [{"names": ["changes_requested"], "roles": ["depositing"]}],
```

Comment:
The state above is called `changes_required` and here it's `changes_requested`. Change this one to `changes_required`?